components/SortingSelector.jsx

```diff
@@ -0,0 +1,46 @@
+import styled from "styled-components";
+
+export default function SortingSelector({ setSortingOrder }) {
+  function handleSortingOrder(event) {
+    setSortingOrder(event.target.value);
+  }
+
+  return (
+    <StyledDiv>
+      <select name="sortingOrder" defaultValue="" onChange={handleSortingOrder}>
```

Comment:
I would pass the value of the current sortingOrder to be the defaultValue.
That way this value is displayed whenever a user goes to a detail view and back to the list page by clicking on the back button.

```suggestion
      <select name="sortingOrder" defaultValue={sortingOrder} onChange={handleSortingOrder}>
```